src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom_nonce.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long,too-many-arguments
+
+from azure.cli.core.profiles import get_api_version, get_versioned_models
+from azure.cli.core.profiles._shared import ResourceType
+
+from azure.cli.command_modules.storage._factory import storage_client_factory
+
+
+if get_api_version(ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS) in ['2016-12-01']:
+
+    def create_storage_account(resource_group_name, account_name, sku, location,
+                               kind=get_versioned_models(ResourceType.MGMT_STORAGE_STORAGE_ACCOUNTS, model='Kind').storage.value, tags=None, custom_domain=None,
```

Comment:
This is another example of this syntax not being workable for practical purposes.  Everyone is just going to put a string here...